public/js/pimcore/object/helpers/gridTabAbstract.js

```diff
@@ -329,18 +298,56 @@ pimcore.object.helpers.gridTabAbstract = Class.create({
 
         this.buildColumnConfigMenu();
 
+        let items = [
+            this.languageInfo, "-",
+            this.toolbarFilterInfo,
+            this.clearFilterButton, "->",
+            this.selectObjectType, "-",
+            this.checkboxOnlyDirectChildren, "-",
+            this.exportButton, "-",
+            this.columnConfigButton,
+            this.saveColumnConfigButton
+        ];
+
+        if(pimcore.helpers.hasSearchImplementation()) {
+            this.searchField = new Ext.form.TextField(
```

Comment:
LGTM, just leaving a note that we have also to pick the changes in this js file to v1.4 as bug fix.
Otherwise, the combination of pimcore 11.2 without Simple Search bundle installed would still show this search field and crash due to search_backend_data table being not existing when typing something to search

Reply:
Resolved: I've created a PR and merged
https://github.com/pimcore/admin-ui-classic-bundle/pull/546

Reply:
Great, thanks for checking.